lib/index.js

```diff
@@ -90,19 +73,25 @@ bedrock.events.on('bedrock-cli.ready', () => {
   }
 });
 
+function _createUrl(config) {
+  let url = 'mongodb://';
+  if(config.username) {
+    url += `${config.username}:${config.password}@`;
+  }
+  url += `${config.host}:${config.port}/${config.name}`;
+  // this needs to come last
+  if(config.username) {
+    url +=
+      `?authSource=${config.connectOptions.authSource || 'admin'}`;
```

Comment:
Should we just default `connectionOptions.authSource` to `admin` and not have this conditional?

Reply:
We would be specifying auth credentials even when the user is not necessarily using auth. This has led to problems in the past such as setting `username` and `password` in `config.js`. I think it's better to ensure the user sets their `authSource` themselves especially as `admin` is usually the `authSource` only in development. In production you usually have a dedicated database with a different name that holds your mongo users. So this code only runs if a `username` was specified and allows developers running locally to be lazy and not necessarily specify the `authSource`. So I guess in short it is better if `authSource` is set to default in general, if there is a username then ok we need to set it.